package.json

```diff
@@ -28,39 +27,38 @@
     "stylelint-prettier": "^1.1.2",
     "typescript": "^4.4.3"
   },
+  "engines": {
+    "node": "16.14.2",
+    "pnpm": "7.25.1"
+  },
   "scripts": {
     "prepare": "husky install",
-    "preinstall": "npm run check:node",
-    "prebootstrap:install": "npm run check:node",
-    "bootstrap": "npm run bootstrap:install -- --ci",
-    "bootstrap:prod": "npm run bootstrap:install -- --ci -- --only=prod",
-    "bootstrap:install": "lerna bootstrap --force-local",
-    "clean": "lerna clean -y",
-    "check:node": "check-node-version --node 16 --npm 8",
-    "fix": "npm run fix:js && npm run fix:style",
+    "clean": "rm -rf node_modules && pnpm -r --parallel exec rm -rf node_modules",
+    "fix": "pnpm fix:js && pnpm fix:style",
     "fix:js": "eslint --fix --ext .ts,.tsx,.js,.jsx src",
     "fix:style": "stylelint --fix src/**/*.{scss,css}",
     "format": "prettier --write \"src/**/*.{js,jsx,ts,tsx,css,scss,html}\"",
     "format:check": "prettier --list-different \"src/**/*.{js,jsx,ts,tsx,css,scss,html}\"",
     "format:robot": "python3 -m pipenv run robotidy --config tests/robot-tests/robotidy.toml tests/robot-tests",
-    "format:robot:check": "npm run format:robot -- --check --diff",
+    "format:robot:check": "pnpm format:robot -- --check --diff",
     "format:python": "python3 -m pipenv run flake8 tests && python3 -m pipenv run black tests && python3 -m pipenv run isort tests",
     "format:python:check": "python3 -m pipenv run flake8 tests --diff && python3 -m pipenv run black tests --diff && python3 -m pipenv run isort tests --diff",
-    "lint": "npm run lint:js && npm run lint:style",
+    "lint": "pnpm lint:js && pnpm lint:style",
     "lint:js": "eslint --ext .ts,.tsx,.js,.jsx src",
     "lint:style": "stylelint src/**/*.{scss,css}",
-    "start:admin": "cd src/explore-education-statistics-admin && npm start",
+    "start:admin": "cd src/explore-education-statistics-admin && pnpm start",
     "start:azure": "cd \"C:\\Program Files (x86)\\Microsoft SDKs\\Azure\\Storage Emulator\" && c:AzureStorageEmulator.exe start -inprocess",
     "start:docker:db": "cd src && docker-compose up db",
-    "start:frontend": "cd src/explore-education-statistics-frontend && npm run start:local",
+    "start:frontend": "cd src/explore-education-statistics-frontend && pnpm start:local",
     "start:processor": "cd src/GovUk.Education.ExploreEducationStatistics.Data.Processor && func start --build",
-    "test": "lerna run test",
-    "test:snapshot": "lerna run test:snapshot",
-    "test:ci": "lerna run test:ci",
-    "test:coverage": "lerna run test:coverage",
-    "tsc": "lerna run tsc",
+    "test": "pnpm -r test",
```

Comment:
Annoyingly, it seems like `pnpm -r` strips out all the ANSI colour codes. I end up with this naff looking result:

![image](https://user-images.githubusercontent.com/9917868/214113625-b08cff48-4628-4b1b-b68f-f2bcdfd353ed.png)

Unfortunately, it seems like this is an existing issue: https://github.com/pnpm/pnpm/issues/2148

I guess this will do for the time being, but I think this is something that NX will do better when we introduce it.